hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/resource/PrisonersInPrisonResource.kt

```diff
@@ -127,20 +134,32 @@ class PrisonersInPrisonResource(private val searchService: PrisonersInPrisonServ
     @RequestParam(value = "incentiveLevelCode", required = false)
     @Parameter(description = "Filter for the prisoners on an incentive level.", example = "STD")
     incentiveLevelCode: String?,
+    @RequestParam(value = "requestedAttributes", required = false)
```

Comment:
not a great fan of this name.  It is a parameter on the request but actually operates on the response, hence including `requested` could lead to confusion.  Maybe `responseAttributes` would be better?
It is interesting that you are calling them attributes too.  Since it is json that is being returned I would have referred to them as fields or properties in the json rather than attributes of the json.

Reply:
Or maybe `responseFields` is actually better because then it's distinct from attribute search and might avoid confusion. They are 2 different thing after all, just sort of related 🤔